java/src/main/java/org/wildfly/openssl/DefaultByteBufferPool.java

```diff
@@ -38,10 +38,12 @@
 class DefaultByteBufferPool {
 
     //TODO: make configurable
+    static final DefaultByteBufferPool WRITE_DIRECT_POOL = new DefaultByteBufferPool(true, Integer.getInteger("org.wildfly.openssl.buffer-size", 16 * 1024));
+    static final DefaultByteBufferPool WRITE_HEAP_POOL = new DefaultByteBufferPool(false, Integer.getInteger("org.wildfly.openssl.buffer-size", 17 * 1024));
```

Comment:
Why do you need the second heap pool?

Reply:
To separate between read/write buffer. I see a TODO to make it configurable. What would you suggest ?

Reply:
@stuartwdouglas  - What would you suggest?

Reply:
I don't think you need two different pools, as that can potentially use more memory that is required. I don't think that todo is actually worth doing, as the size can already be confgured from a system prop.

Reply:
@stuartwdouglas  - Got it, Will remove the additional WRITE_HEAP_POOL and for write direct pool configuration rename to "org.wildfly.openssl.write-buffer-size".